ChangeLog.rst

```diff
@@ -2,6 +2,11 @@
  ChangeLog
 ===========
 
+1.27.12 (2025-09-26)
```

Comment:
In semantic versioning there are major.minor.patch parts of a version. Patch part is incremented on bug fixes. But I've added a feature. It would be more correct to use 1.28.0 version for this release.